.github/workflows/backport-command.yml

```diff
@@ -0,0 +1,61 @@
+name: backport-command
+on:
+  repository_dispatch:
+    types: [backport-command]
+jobs:
+  backport:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install gh cli
+        run: |
+          curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+          sudo apt update
+          sudo apt install gh
+          gh auth login --with-token <<< "${{ secrets.ACTIONS_BOT_TOKEN }}"
+
+      - name: Add reaction
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token: ${{ secrets.ACTIONS_BOT_TOKEN }}
+          repository: ${{ github.event.client_payload.github.payload.repository.full_name }}
+          comment-id: ${{ github.event.client_payload.github.payload.comment.id }}
+          reaction-type: hooray
+
+      - name: Get assignees
+        env:
+          ASSIGNEES: ${{ toJson(github.event.client_payload.github.payload.issue.assignees) }}
+        id: assignees
+        run: echo ::set-output name=assignees::$(echo "$ASSIGNEES" | jq -r '.[].login' | paste -s -d ',' -)
```

Comment:
should probably move this to the end of the workflow so it only adds a reaction if the issue creation completed successfully